File: app/src/main/java/com/laushkina/anastasia/psychosophy/domain/test/AnswersValidation.java
package com.laushkina.anastasia.psychosophy.domain.test;

public class AnswersValidation {

    public static boolean areValid(TestQuestion[] questions) {
        boolean isValid = true;
        for (TestQuestion question : questions) {
            if (question.getAnswer() == null) {
                isValid = false;
                break;
            }
        }
        return isValid;
    }
}

File: app/src/main/java/com/laushkina/anastasia/psychosophy/domain/test/AnswersValidation.kt
package com.laushkina.anastasia.psychosophy.domain.test

class AnswersValidation {
    companion object {
        fun areValid(questions: Array<TestQuestion?>): Boolean {
            var isValid = true
            for (question in questions) {
                if (question?.answer == null) {
                    isValid = false
                    break
                }
            }
            return isValid
        }
    }
}